changelog.md

```diff
@@ -1,7 +1,10 @@
 ## Changes from 1.2.0 to (unreleased version)
 
+## Changes from 1.1.0 to 1.3.0
```

Comment:
Can you please move the `Changes from 1.2` below 1.3? The latest release version should be the top entry, as in

`1.1.0 to 1.3.0`
`1.2.0 skipped`
`1.0.0 to 1.1.0`